test/helper.js

```diff
@@ -33,7 +33,7 @@ if (!cfg.get('azure:accountKey') ||
 var webServer = null;
 mocha.before(async () => {
   webServer = await server('test');
-  webServer.setTimeout(500);
+  webServer.setTimeout(5000);
```

Comment:
hmm... Can we try with `1500` maybe that works better with nodes keepAlive setting...